lib/broccoli/default-packager.js

```diff
@@ -12,6 +12,7 @@ const ConfigReplace = require('broccoli-config-replace');
 const emberAppUtils = require('../utilities/ember-app-utils');
```

Comment:
TODO: copy the logic from embroider, which is _way_ simpler: https://github.com/embroider-build/embroider/blob/9b83c1139f53055c57dc51c7321c19fe8bf31974/packages/core/src/app.ts#L309